tests/rptest/tests/schema_registry_test.py

```diff
@@ -4130,6 +4130,144 @@ def test_mode_readonly(self):
         result_raw = self.sr_client.get_schemas_ids_id_versions(id=1)
         assert result_raw.status_code == 200
 
+    @cluster(num_nodes=3)
```

Comment:
Is it worth adding a check that with global IMPORT, subject READ_WRITE behaves as intended (override?)?